frontend/src/pages/AnnouncementPage.js

```diff
@@ -119,88 +130,163 @@ function AnnouncementEntry({
 		WebkitLineClamp: 2,
 	} : {};
 
-	const boxSidebarStyle = { display: "flex", mt: 1, mb: 1, pb: 1, borderBottom: 1, borderColor: "grey.800" };
-	const boxPageStyle = { display: "flex", mt: 1, mb: 3, p: 1, border: 1, borderColor: "grey.800" };
+	const boxPageStyle = { display: "flex", mt: 1, mb: 3, p: 1, border: 1, borderColor: "rgba(255, 255, 255, 0.1)" };
 	const titleStyle = { fontFamily: "Montserrat", fontSize: "14px", fontStyle: "normal", fontWeight: "bold" };
-	const dateStyle = { 
-		fontFamily: "Noto Sans", fontSize: "11px", fontStyle: "normal", fontWeight: "medium", marginBottom: 3, opacity: 0.5
+	const dateStyle = {
+		fontFamily: "Noto Sans", fontSize: "11px", fontStyle: "normal",
+		fontWeight: "medium", opacity: 0.4
 	};
-	const contentStyle = { fontFamily: "Noto Sans", fontSize: "12px", fontStyle: "normal", fontWeight: "medium" };
+	const contentStyle = {
+		fontFamily: "Noto Sans", fontSize: "12px",
+		fontStyle: "normal", fontWeight: "medium",
+		color: "rgba(255, 255, 255, 0.81)"
+	};
+	const randomPhoto = Photos[Math.floor(Math.random() * Photos.length)];
```

Comment:
Please remove the randomness here. When I change the screen size it will randomly change images which is not a good thing for a product presentation.

Reply:
I did this:
```suggestion
	const announcePhoto = photos[parseInt(announcement.$id, 16) % photos.length];
```
I changed the names to `photos` and `announcePhoto` (via Renaming feature in VSCode).